pkgs/by-name/ta/taskflow/package.nix

```
@@ -35,6 +35,11 @@ stdenv.mkDerivation rec {
     cmake
   ];
 
+  cmakeFlags = [
+    # building the tests implies running them in the buildPhase
+    (lib.cmakeBool "TF_BUILD_TESTS" (stdenv.buildPlatform.canExecute stdenv.hostPlatform))
```

```suggestion
    (lib.cmakeBool "TF_BUILD_TESTS" finalAttrs.finalPackage.doCheck)
```

where `finalAttrs` comes from changing the derivation to be `stdenv.mkDerivation (finalAttrs: { ... })` instead of `stdenv.mkDerivation rec { ... }`

aside from that, everything builds & looks fine 👍